### ISSUE TITLE
Creating Loadouts from Loadout Builder Doesn't Default to Equipped

### ISSUE THREAD

**Issue by SunburnedGoose** [DestinyItemManager/DIM#750]
When a loadout is created from the Loadout Builder, it should mark all items as 'Equip'.

![screen shot 2016-08-05 at 2 06 24 pm](https://cloud.githubusercontent.com/assets/984608/17447749/dd7d150a-5b15-11e6-9ee5-7a9b9223ff48.png)

**Comment by SunburnedGoose** [comment]
@ericnelson0, is this your area of concern?

**Comment by ericnelson0** [comment]
Yeah, I'll look into it.

**Comment by SunburnedGoose** [comment]
It's pretty easy.  When you create the clone, just mark it as equipped I believe.

**Comment by SunburnedGoose** [comment]
I just removed the watch because it wasn't performing any useful function.  The idea was if the loadout was modified beyond the scope of the loadout directive, then it would update with the latest data.  We don't do that afaik, so I removed it.  

Before I make a PR, I want to consider something.  Only scenarios I can consider is if sync occurs in the background while the loadout was open.  How does this data get integrated into loadouts, and synced into the directive?
